UPGRADE-3.0.md

```diff
@@ -1,6 +1,35 @@
 UPGRADE FROM 2.x to 3.0
 =======================
 
+# Table of Contents
+
+- [ClassLoader](#classloader)
+- [Config](#config)
+- [Console](#console)
+- [DependencyInjection](#dependencyinjection)
+- [DomCrawler](#domcrawler)
+- [DoctrineBridge](#doctrinebridge)
+- [EventDispatcher](#eventdispatcher)
+- [Form](#form)
+- [FrameworkBundle](#frameworkbundle)
+- [HttpKernel](#httpkernel)
+- [Locale](#locale)
+- [PropertyAccess](#propertyaccess)
+- [Routing](#routing)
+- [Security](#security)
+- [SecurityBundle](#securitybundle)
+- [Serializer](#serializer)
+- [Translator](#translator)
+- [Twig Bridge](#twig-bridge)
+- [TwigBundle](#twigbundle)
+- [Validator](#validator)
+- [Yaml](#yaml)
+- [WebProfiler](#webprofiler)
+- [Process](#process)
+- [Monolog Bridge](#monolog-bridge)
+- [Swiftmailer Bridge](#swiftmailer-bridge)
+- [HttpFoundation](#httpfoundation)
```

Comment:
imo this should be sorted alphabetically to make it easier to find what you are looking for